frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/vm/advanced-wizard.ts

```diff
@@ -87,7 +87,7 @@ describe('Test VM creation', () => {
     });
   });
 
-  [urlVM, registryVM, pvcVM, pxeVM].forEach((vmData) => {
+  [urlVM, registryVM, pvcVM].forEach((vmData) => {
```

Comment:
NOTE:
this is skipping the PXE test, we will need to re-introduce this test once we fix the test flakynes
I don't think the PXE test flakynes is connected to this PR